public/app/features/inspector/InspectJSONTab.tsx

```diff
@@ -1,11 +1,12 @@
 import { t } from '@lingui/macro';
+import { isEqual } from 'lodash';
```

Comment:
Out of curiosity, why do we use the entire lodash instead of the partial packages `lodash.isequal`?

Reply:
🤔 good question, I don't really have the answer 😛, I've seen it used that way in other places but I'll give it a try and use your suggestion 👀

Reply:
I was investigating a bit about why we don't use this and found that [per method packages](https://lodash.com/per-method-packages) are discouraged, and it will be removed in v5, so probably that's the reason 😄